### DISCUSSION TITLE
Feature to force enable fans when temperature limit is reached

### DISCUSSION THREAD

**Discussion by devklepacki** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1016] [Ideas]
Another thought regarding the overheating 😁 It's a very hot day here and I've just installed Mac Fan Control to manually enable fans to cool down the battery. It's a far-fetched idea but it'd maybe be something to be added to Aldente? Like a single setting to automatically enable fans if a device is plugged in and the temperature limit is reached?

And an additional thing regarding fans, if the previous setting was implemented, maybe additionally add a setting for max RPM limit so the fans are not at full blast.

**Comment by MatthiasKerbl** [comment]
Hi @devklepacki,

excellent idea. There is already a feature request open for this #630, therefore I will close the discussion here.

Best,
Matthias

**Reply by devklepacki** [reply]
Thank you! I searched for this but now I see I was searching for the wrong keywords in both cases.
